superset-frontend/cypress-base/cypress/integration/chart_list/list.test.ts

```diff
@@ -68,6 +68,20 @@ describe('Charts list', () => {
       cy.createSampleCharts([0]);
     });
 
+    after(() => {
+      cy.visit(DASHBOARD_LIST);
+      cy.wait(1000);
+      toggleBulkSelect();
+      cy.wait(1000);
+      cy.getBySel('table-row').each($row => {
+        if ($row.text().indexOf('Sample dashboard') > -1) {
+          cy.wrap($row).find('input[type="checkbox"]').click();
+        }
+      });
+      cy.getBySel('bulk-select-action').eq(0).contains('Delete').click();
+      confirmDelete();
+    });
+
```

Comment:
@codyml There is a utils that it seems to do this is `cy.deleteDashboard(id: number, failOnStatusCode: boolean)` or `deleteDashboardByName(dashboardName: string, failOnStatusCode: boolean)`.